betfairlightweight/baseclient.py

```diff
@@ -156,9 +156,8 @@ def cert(self) -> list:
             return self.cert_files
 
         certs = self.certs or "/certs/"
```

Comment:
I am thinking that maybe we should update the class initialisation as well and remove this line:

`certs: str = os.path.join(“certs”)`